.github/workflows/build-and-test-windows.yml

```diff
@@ -89,3 +89,24 @@ jobs:
             echo "One or more matrix jobs failed."
             false
           fi
+  integration-tests:
```

Comment:
How can we get a confirmation that this is working? I feel like there should be a flag being set somewhere. Or is goenv enough?

Reply:
Is there any way to test it? For the flag are you talking about a flag to confirm the execution?

Reply:
Sorry, I meant whether simply running the tests on Windows would activate the right tests to be executed. I assume this is the case, but it would be nice to get positive confirmation.

Reply:
Oh okay, I got you I going to test it

Reply:
I tried to test it on my windows machine, and it's failling due to 
` runtime/cgo  cc1.exe: sorry, unimplemented: 64-bit mode not compiled in  FAIL    github.com/open-telemetry/opentelemetry-collectorcontrib/extension/observer/dockerobserver [build failed]   FAIL  `. The tests are found but for them to process it's failling. I don't know if it's my setup or a setting I forgot to set

#### Environment: Windows-11 with (MINGW64 bash emulator)

![Screenshot 2023-04-10 212540](https://user-images.githubusercontent.com/82499435/230981775-47dfa449-9158-4884-ba81-4c82e74df328.png)

Reply:
Sorry, I haven't seen this one before: perhaps it's specific to Windows/